editor/js/Loader.js

```diff
@@ -858,16 +858,36 @@ function Loader( editor ) {
 
 		const zip = unzipSync( new Uint8Array( contents ) );
 
+		const manager = new THREE.LoadingManager();
+		manager.setURLModifier( function ( url ) {
+
+			const u8a = zip[ url ];
+
+			if ( u8a ) {
+
+				console.log( 'Loading', url );
+
+				const blob = new Blob( [ u8a ], { type: 'application/octet-stream' } );
```

Comment:
Can you please retain the `file` variable and also `file.buffer`. I have tested it locally and there should be no need to change the URL modifier. 

It is also sufficient to just add it to `MTLLoader`.  `OBJLoader` does not need the manager.

Reply:
So it should be:
```js
const manager = new THREE.LoadingManager();
manager.setURLModifier( function ( url ) {

	const file = zip[ url ];

	if ( file ) {

		console.log( 'Loading', url );

		const blob = new Blob( [ file.buffer ], { type: 'application/octet-stream' } );
		return URL.createObjectURL( blob );

	}

	return url;

} );
```

Reply:
good.